openerp/fields.py

```diff
@@ -948,6 +1029,8 @@ def modified_draft(self, records):
 class Boolean(Field):
     type = 'boolean'
 
+    __slots__ = []
```

Comment:
Maybe that could be set in MetaField.__new__? That way there's no need to specify it if you don't want slots. In fact maybe that could use a descriptor shortcut to avoid all the `_default_{thing}` gunk (and even possibly the _related_ ones)